packages/playground/src/components/project-settings/form-member-delete-dialog.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react'
+import {
+  AlertDialog,
+  AlertDialogTrigger,
+  AlertDialogContent,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogCancel,
+  Button,
+  Icon,
+  Spacer
+} from '@harnessio/canary'
+
+interface MemberProps {
+  display_name?: string
+}
+interface FormDeleteMemberDialogProps {
+  member: MemberProps
+  onClose: () => void
+}
+
+export const FormDeleteMemberDialog: React.FC<FormDeleteMemberDialogProps> = ({ member, onClose }) => {
+  const [isDeleting, setIsDeleting] = useState(false)
+  const [deleteSuccess, setDeleteSuccess] = useState(false)
+  // Delete project handler
+  const handleDelete = () => {
+    setIsDeleting(true)
+    setTimeout(() => {
+      setIsDeleting(false)
+      setDeleteSuccess(true) // Mark deletion as successful
+      setTimeout(() => {
+        onClose() // Close the dialog
+      }, 2000)
+    }, 2000)
+  }
```

Comment:
I think these timeouts need to come in as props as part of the callback fn from the parent playground page